src/__tests__/HomePageTests/GetStartedCard.test.tsx

```diff
@@ -4,19 +4,43 @@ import GetStartedCard from "../../components/Home/Sections/GetStarted/GetStarted
 
 describe("GetStartedCard component test", () => {
     test("GetStartedCard component is rendered", () => {
-        const result = render(<GetStartedCard number='1' href='https://deriv.com/' icon='test.svg' title='test title' content='test content' />);
+        const result = render(
+            <GetStartedCard
+                number="1"
+                href="https://deriv.com/"
+                icon="test.svg"
+                title="test title"
+                content="test content"
+            />
+        );
         expect(result.container.querySelector(".header")).toBeInTheDocument();
         expect(result.container.querySelector(".hide-on-mobile")).toBeInTheDocument();
         expect(result.container.querySelector(".hide-on-desktop")).toBeInTheDocument();
     });
 
     test("GetStartedCard component has '1. test title' text", () => {
-        render(<GetStartedCard number='1' href='https://deriv.com/' icon='test.svg' title='test title' content='test content' />);
+        render(
+            <GetStartedCard
+                number="1"
+                href="https://deriv.com/"
+                icon="test.svg"
+                title="test title"
+                content="test content"
+            />
+        );
         expect(screen.getByText("1. test title")).toBeInTheDocument();
     });
 
     test("GetStartedCard component has 'test icon' altText", () => {
-        render(<GetStartedCard number='1' href='https://deriv.com/' icon='test.svg' title='test title' content='test content' />);
+        render(
+            <GetStartedCard
+                number="1"
+                href="https://deriv.com/"
+                icon="test.svg"
+                title="test title"
+                content="test content"
+            />
+        );
         expect(screen.getByAltText("test icon")).toBeInTheDocument();
```

Comment:
don't repeat yourself, you can create a variable props, and then spread it to each component:
const {
number="1"
href="https://deriv.com/"
icon="test.svg"
title="test title"
content="test content"
} = props

 <GetStartedCard {...props} />

Reply:
This message is for Henadziy! Not for Maria !! 😊